10.9-libcxx/stable/main/finkinfo/languages/python38.info

```diff
@@ -37,7 +37,7 @@ BuildDepends: <<
 	readline8,
 	sqlite3-dev (>= 3.30.1-1),
 	openssl300-dev (>= 3.0.1-1),
-	pkgconfig (>= 0.9.0-1),
+	pkgconf,
```

Comment:
ppkg-config? And fix $PKG_CONFIG down below to match?

Reply:
Have not tested this, but `python310` did not report any errors of the kind in https://github.com/fink/fink-distributions/pull/996#issuecomment-1430617016, so it's probably safe to switch as well.